cubic-server/world-storage/WorldStorage.hpp

```diff
@@ -0,0 +1,29 @@
+#include <unordered_map>
+#include "LevelData.hpp"
+#include "Level.hpp"
+
+#ifndef WORLD_STORAGE_HPP
+#define WORLD_STORAGE_HPP
+
+class WorldStorage
+{
+public:
+    WorldStorage();
+    ~WorldStorage();
+
+    const LevelData &getLevelData() const;
+    void setLevelData(const LevelData &value);
+
+    const std::vector<std::string> getLevelNames() const;
+    const Level *getLevel(std::string levelName) const;
+    Level *getLevelM(std::string levelName);
```

Comment:
Please use a constant reference for arguments with objects, especially when it is a `std::string`

Reply:
Also please rename this method, for example `getEditableLevel`